napari/_vispy/_tests/test_vispy_surface_layer.py

```diff
@@ -125,7 +127,8 @@ def check_surface_without_visible_faces():
     )
     faces = np.array([[0, 1, 2], [3, 4, 5]])
     layer = Surface((points, faces))
-    viewer = napari.Viewer(ndisplay=3)
+    viewer = make_napari_viewer(ndisplay=3)
+    viewer.show()
```

Comment:
Add comment that this is here to trigger something?

Reply:
Ok, it's good to just have it here actually, this is a test so we shouldn't need to touch it much.